addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -315,6 +316,80 @@
             </field>
         </record>
 
+        <!-- Inherited form view for mass mailing's form specifically -->
+        <record model="ir.ui.view" id="mailing_mailing_view_full_width_form">
+            <field name="name">mailing.mailing.form_inherit</field>
+            <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_form"/>
+            <field name="mode">primary</field>
```

Comment:
Well we should probably force its priority to ensure it is not used as default view, as it is used only in email marketing context (otherwise fallback on standard view, unless I misunderstood something)

Reply:
I'm not sure what you mean. Do you mean I should add like `<field name="priority">2</field>` on this one and `<field name="priority">1</field>` on the other ?